config.yml

```diff
@@ -0,0 +1,15 @@
+experiment_name: simple_16
+n_repeats: 10
+scenario_params_list:
+ - nodes_counts: 3
+   amounts_per_node: [0.4, 0.3, 0.3] 
+   samples_split_option: 'Random'
+   testset_option: 'Centralised'
+ - nodes_counts: 3
+   amounts_per_node: [0.9, 0.05, 0.05] 
+   samples_split_option: 'Random'
+   testset_option: 'Centralised'
+ - nodes_counts: 2
+   amounts_per_node: [0.9, 0.1] 
+   samples_split_option: 'Random'
+   testset_option: 'Centralised'
```

Comment:
Can you please add a new blank line at the end of the document (to reduce the diff)?